dags/sql/select-sensor_id_count_lookback.sql

```diff
@@ -0,0 +1,60 @@
+--query to be used in `check_row_count` tasks to compare
+--row count to average over lookback period.
+--noqa: disable=TMP, PRS
+
+WITH lookback AS ( --noqa: L045
+    SELECT
+        date_trunc('day', {{ params.dt_col }}) AS _dt, --noqa: L039
+        COUNT(DISTINCT {{ params.id_col }}) AS lookback_count, --noqa: L039
+        ARRAY_AGG(DISTINCT {{ params.id_col }}) AS daily_ids --noqa: L039
+    FROM {{ params.table }}
+    WHERE
+        {{ params.dt_col }} >= '{{ ds }} 00:00:00'::timestamp - interval '{{ params.lookback }}'
+        AND {{ params.dt_col }} < '{{ ds }} 00:00:00'::timestamp
+    --group by day then avg excludes missing days.
+    GROUP BY _dt --noqa: L003
+),
+
+today AS (
```

Comment:
Can we change this to selecting from lookback as well? It is ever so slighter faster this way.